### ISSUE TITLE
[NPM] Error using some packages

### ISSUE THREAD

**Issue by spyl94** [brunch/brunch#1108]
Just reporting some errors, I will try to investigate more:

- [x] react-bootstrap (^0.28.2)

Compiling: :+1: 
But some modules are in `node_modules` but not found when running the application code:
```
Uncaught Error: Cannot find module "react-prop-types/lib/singlePropFrom" from "react-bootstrap/utils/childrenValueInputValidation"
```
Code: https://github.com/spyl94/react-brunch-demo/tree/add-react-bootstrap

- [ ] react-intl (^2.0.0-beta-2)

Compiling: :-1: 
````
/Users/aurel/Web/react-brunch-demo/node_modules/brunch/lib/deppack.js:5
const browserResolve = require('browser-resolve');
                                               ^

TypeError: Path must be a string. Received false
  at assertPath (path.js:8:11)
  at Object.posix.join (path.js:479:5)
  at relativeToRoot (/Users/aurel/Web/react-brunch-demo/node_modules/brunch/lib/deppack.js:50:55)
  at /Users/aurel/Web/react-brunch-demo/node_modules/brunch/lib/deppack.js:75:22
  at Array.forEach (native)
  at Object.keys.filter.forEach (/Users/aurel/Web/react-brunch-demo/node_modules/brunch/lib/deppack.js:73:41)
  at generateModule (/Users/aurel/Web/react-brunch-demo/node_modules/brunch/lib/deppack.js:99:24)
  at /Users/aurel/Web/react-brunch-demo/node_modules/brunch/lib/deppack.js:151:13
  at tryToString (fs.js:414:3)
  at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:401:12)
```
Code: https://github.com/spyl94/react-brunch-demo/tree/add-intl

I'm using node v5.1.1 and npm 3.5.3.

**Comment by paulmillr** [comment]
Could you try using the version from GitHub?

```
npm install -g brunch/brunch
```

**Comment by spyl94** [comment]
Using master fixed react-bootstrap :+1: 

But support of shims seems to be broken:
```
Uncaught Error: Cannot find module "events" from "js/stores/BaseStore"`
```
`npm install --save events` and removing `events` from emptyShims in deppack.js can solve the problem.

Compillation of react-intl (^2.0.0-beta-2) is still failing :(

**Comment by spyl94** [comment]
The problem with `react-intl` is the usage of:
```
"browser": {
    "./lib/locales": false,
    "./lib/locales.js": false
  },
```
in their `package.json`, removing it fix the compilation issue.

**Comment by kyledecot** [comment]
I'm having a similar issue when attempting to use `react-router`. See:

http://stackoverflow.com/questions/34823964/using-react-router-w-brunch-babel

**Comment by paulmillr** [comment]
@goshakkk

**Comment by kyledecot** [comment]
I just pulled in the latest brunch from github and I'm still experiencing this.

**Comment by goshakkk** [comment]
@spyl94 I've opened a PR with fixes to make your react intl sample work (#1116). Note that for the time being, you're still going to add `events` to your `package.json`. A better shims solution is being designed and implemented.

**Comment by goshakkk** [comment]
@kyledecot reporting bugs like that via SO is unhelpful. Do you mind opening a separate ticket for that and including some more details, at least the React Router version you're using; ideally a small sample app that triggers the issues?

**Comment by kyledecot** [comment]
@goshakkk at first it was a question for SO as I thought I was just doing something wrong. Only after seeing this issue did I realize that it seemed to be connected. I'll open a new ticket w/ all the relevant details.
